src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { Checkbox } from "./Checkbox";
+import { TextArea } from "../TextArea/TextArea";
+
+export default {
+  title: "USWDS/Base/Checkbox",
+  component: Checkbox,
+  argTypes: {
+    id: {
+      description: "A unique identifier for the input.",
+    },
+    isTile: {
+      description: "Use the tile variation of the Checkbox.",
+    },
+    label: {
+      description: "Label text that appears to the right of the Checkbox.",
+    },
+    name: {
+      description: "Name of the input element.",
+    },
+    tileDescription: {
+      description:
+        "Text that can be used to describe the label in more detail. Activates the tile variation automatically.",
+    },
+    value: {
+      description: "Value of the input element.",
+    },
+  },
+  args: {
+    checked: false,
+    isTile: false,
+    tileDescription: "",
+  },
```

Comment:
For future reference looks like you can pop in the label property here and it'll populate it in every example. Not an issue either way
`label: "Checkbox Item"`

Reply:
Nice! I've refactored it to do it this way. Great call-out, thank you